### ISSUE TITLE
Comments truncated

### ISSUE THREAD

**Issue by ToonTalk** [jmoenig/Snap--Build-Your-Own-Blocks#1836]
Starting with

![image](https://user-images.githubusercontent.com/2138696/29554431-697c1af2-871f-11e7-9e60-3bc959bfa668.png)

If I make it one line it looks like this:

![image](https://user-images.githubusercontent.com/2138696/29554458-850cd84c-871f-11e7-8490-8803cab0ce7e.png)

Dev version on Chrome 60, Windows 10 64-bit.

**Comment by brianharvey** [comment]
Yeah, the boldface font is a little wider than the lightface one.  Resize the box a little wider before going to single-line mode.

**Comment by ToonTalk** [comment]
Why not use the same font in both cases? [WYSIWYG](https://en.wikipedia.org/wiki/WYSIWYG)

**Comment by ToonTalk** [comment]
Also why should multi-line comments use a different font style than single line ones? (And that single line ones can be edited only by temporarily changing them to multi-line?)

**Comment by cycomachead** [comment]
> Also why should multi-line comments use a different font style than single line ones? (And that single line ones can be edited only by temporarily changing them to multi-line?)

The interpretation (I believe) isn't multi- vs single- line, it's more of a title and description. 

Collapsing a comment is to hide details. And, while this is a design inherited from Scratch, I believe the reason for bold was to make the title stand out a bit more. However, the Snap<em>!</em> backgrounds darker than Scratch's so we could revisit that.

**Comment by brianharvey** [comment]
What?  No, it's not a title, because you can't have one if you are in expanded mode.  It's the same text as the expanded comment, only in a smaller box and a different font.

**Comment by cycomachead** [comment]
Oh well, it _used_ to be that way in BYOB, though I guess I never picked up on the behavior in Snap<em>!</em>.

In BYOB (and Scratch 1 and 2), only the first line was shown when the comment is collapsed. Snap<em>!</em> just concatenates the text which I do think is wrong. They would also indicate a truncated comment with an ellipsis, which Snap<em>!</em> should do too. (Both suffered from the same issue that the triangle toggle took up more room, but this seems acceptable to me. Otherwise we could just extend the collapse state by a little bit.)

In any event, I think the old behavior of comments was more clear.

**Comment by brianharvey** [comment]
Oh, I see, maybe you're right about only the first line showing when collapsed.  I agree, that would be good.  And so would extending the width just enough to make the comment still fit, or perhaps using a slightly condensed boldface font so that the space is the same.
